src/components/Toast/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import { TOAST_SKIN } from './Toast';
+
+export function getMessage(skin: TOAST_SKIN): string {
```

Comment:
If you use the following code, when a new skin is added a compile error would occur (and then you would have to update this object) instead of having an empty string being returned in run time
```
const skinMessages: {[skin in keyof typeof TOAST_SKIN]: string} = {
  success: '24 photos were uploaded successfully.',
  error: 'These files exceed the upload limit.',
  status: 'Uploading...',
};
```